package.json

```diff
@@ -18,17 +18,15 @@
     "sage",
     "design system"
   ],
+  "type": "module",
   "scripts": {
     "test": "mocha",
     "test:watch": "npm test -- --watch",
-    "build": "node ./scripts/build.js && npm run post-process-css",
-    "prebuild": "node ./scripts/prebuild.js",
-    "postbuild": "node ./scripts/postbuild.js",
-    "post-process-css": "npx ts-node ./scripts/post-process-css",
-    "dev:docs": "nodemon --watch \"./scripts/*\" --watch \"./templates/**/*\" --exec \"node ./scripts/tokens-documentation.js\"",
-    "prepublishOnly": "node ./scripts/bump-main-package-version.js",
-    "lint": "eslint .",
-    "format": "eslint --fix .",
+    "build": "npx tsx ./scripts/build && npm run post-process-css",
+    "prebuild": "npx tsx ./scripts/prebuild",
+    "postbuild": "npx tsx ./scripts/postbuild",
+    "post-process-css": "npx tsx ./scripts/post-process-css",
+    "prepublishOnly": "npx tsx ./scripts/bump-main-package-version",
```

Comment:
I haven't checked this first hand so may not be accurate, but this says that `tsx` doesn't do type checking in favour of performance.
https://www.reddit.com/r/node/comments/16qnlhy/comment/kv9cp9g

Given this is just a build tool, I'd personally favour accuracy over performance.